templates/components/form/form.jsx

```diff
@@ -78,12 +80,11 @@ var Form = React.createClass({
   },
   render: function() {
      var fields = Object.keys(this.props.fields).map(this.buildFormElement);
-     return <div role="form">{fields}</div>;
+     return <div role="form"><form>{fields}</form></div>;
```

Comment:
I just though doing this might be useful in term of applying style to every place as a standard.